src/views/SignIn.vue

```diff
@@ -1,6 +1,23 @@
 <template>
   <div class="sign-in">
-      <h2>Sign In</h2>
+    <div class="container">
+      <div class="card" style="width: 18rem;">
```

Comment:
add "card-sign-in" class, so it can look like 
div class="card card-sign-in" style="width: 18rem;"